Artifacts/MsSql/Structure/Admin/0160-Rename-OdsInstanceContext.sql

```diff
@@ -0,0 +1,41 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+ALTER TABLE [dbo].[OdsInstanceContext]
+DROP CONSTRAINT [FK_OdsInstanceContext_OdsInstanceId_OdsInstanceId];
+
+ALTER TABLE [dbo].[OdsInstanceContext]
+DROP CONSTRAINT [UC_OdsInstanceDerivative_OdsInstanceId_ContextKey];
+    
+EXEC SP_RENAME 'dbo.OdsInstanceContext.OdsInstanceId', 'OdsInstance_OdsInstanceId', 'COLUMN';
```

Comment:
IF EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = 'OdsInstanceContext' AND COLUMN_NAME = 'OdsInstanceId')
BEGIN
    EXEC SP_RENAME 'dbo.OdsInstanceContext.OdsInstanceId', 'OdsInstance_OdsInstanceId', 'COLUMN';
END

Reply:
Here Also before renaming column check exists or not and then rename the column